index.html

```diff
@@ -77,22 +78,37 @@ <h3>Load a URL</h3>
+      <option value="https://devstreaming-cdn.apple.com/videos/streaming/examples/bipbop_adv_example_hevc/master.m3u8">Advanced Bipbop - FMP4 hevc, demuxed</option>
+      <option value="https://storage.googleapis.com/shaka-demo-assets/angel-one-hls/hls.m3u8">Angel One - FMP4 unmuxed, many audio/captions</option>
+      <option value="https://bitdash-a.akamaihd.net/content/MI201109210084_1/m3u8s-fmp4/f08e80da-bf1d-4e3d-8899-f0f6155f6efa.m3u8">Parkour - FMP4 unmuxed</option>
+      <option value="https://s3.amazonaws.com/qa.jwplayer.com/~alex/121628/new_master.m3u8">Song - ts Audio only</option>
+      <option value="https://d2zihajmogu5jn.cloudfront.net/CoitTower/master_ts_segtimes.m3u8">Coit Tower drone footage - 4 8 second segment</option>
+      <option value="https://playertest.longtailvideo.com/adaptive/oceans_aes/oceans_aes.m3u8">Disney's Oceans trailer - HLSe, ts Encrypted</option>
+      <option value="https://bitmovin-a.akamaihd.net/content/sintel/hls/playlist.m3u8">Sintel - ts with audio/subs and a 4k rendtion</option>
+      <option value="https://storage.googleapis.com/shaka-demo-assets/bbb-dark-truths-hls/hls.m3u8">BBB-CMIF: Big Buck Bunny Dark Truths - demuxed, fmp4</option>
     </optgroup>
     <optgroup label="dash">
-      <option value="https://dash.akamaized.net/akamai/bbb_30fps/bbb_30fps.mpd">unmuxed audio/video</option>
-      <option value="https://vm2.dashif.org/livesim/mup_30/testpic_2s/Manifest.mpd">Live</option>
-      <option value="https://dash.akamaized.net/dash264/TestCases/10a/1/iis_forest_short_poem_multi_lang_480p_single_adapt_aaclc_sidx.mpd">Sidx unmuxed, 2 audio</option>
+      <option value="https://dash.akamaized.net/akamai/bbb_30fps/bbb_30fps.mpd">Big Buck Bunny - unmuxed audio/video, includes 4K, burns in frame, pts, resolution, bitrate values</option>
+      <option value="https://storage.googleapis.com/shaka-demo-assets/angel-one/dash.mpd">Angel One - fmp4, webm, subs, alternate audio tracks</option>
+      <option value="https://storage.googleapis.com/shaka-demo-assets/bbb-dark-truths/dash.mpd">BBB-CMIF: Big Buck Bunny Dark Truths - demuxed, fmp4</option>
+      <option value="https://dash.akamaized.net/dash264/TestCases/10a/1/iis_forest_short_poem_multi_lang_480p_single_adapt_aaclc_sidx.mpd">SIDX unmuxed, 2 audio</option>
+      <option value="https://download.tsi.telecom-paristech.fr/gpac/DASH_CONFORMANCE/TelecomParisTech/mp4-onDemand/mp4-onDemand-mpd-AV.mpd">SIDX bipbop-like</option>
+      <option value="https://yt-dash-mse-test.commondatastorage.googleapis.com/media/car-20120827-manifest.mpd">Google self-driving car - SIDX</option>
+      <option value="https://d2zihajmogu5jn.cloudfront.net/sintel_dash/sintel_vod.mpd">Sintel - single rendition</option>
+    </optgroup>
+    <optgroup label="live">
+      <option value="https://akamai-axtest.akamaized.net/routes/lapd-v1-acceptance/www_c4/Manifest.m3u8">HLS - Live - Axinom live stream, may not always be available</option>
```

Comment:
If it's not always available, I think we may be better off not including it.

Reply:
It's still helpful when it's up but want to let people know that if it 404ing it's expected.